tests/migration_test.go

```diff
@@ -2946,6 +2946,99 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				expectFeatureToBeSupportedOnNode(newNode, requiredFeatures)
 			})
 
+			Context("[Serial]node with obsolete host-model cpuModel", func() {
```

Comment:
I am not sure this test should be under `migration_test`

Reply:
Done.